spec/abstractops.html

```diff
@@ -1606,10 +1606,13 @@ <h1>
       <dd>It converts _argument_ to an integer representing its Number value with fractional part truncated (replacing *NaN* with 0), or throws a *RangeError* when that Number value is not finite.</dd>
     </dl>
     <emu-alg>
+      1. If _argument_ is *undefined*, throw a *RangeError* exception.
       1. Let _number_ be ? ToNumber(_argument_).
-      1. If _number_ is *NaN*, return 0.
-      1. If _number_ is *+&infin;*<sub>𝔽</sub> or *-&infin;*<sub>𝔽</sub>, throw a *RangeError* exception.
-      1. Return truncate(ℝ(number)).
+      1. If _number_ is *NaN*, -∞, or +∞, throw a *RangeError* exception.
```

Comment:
```suggestion
      1. If _number_ is *NaN*, *-&infin;*<sub>𝔽</sub>, or *+&infin;*<sub>𝔽</sub>, throw a *RangeError* exception.
```
(we have to distinguish between ±∞ as [extended mathematical values](https://tc39.es/ecma262/#extended-mathematical-value-of), and ±∞<sub>𝔽</sub> as Number values)